### PR TITLE
Add sklpandas

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/sklearn-pandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest
```
Comment (synapticarbors) [inline recipes/sklearn-pandas/meta.yaml:21]: I don't think you need `pytest` under `host`.

File: recipes/sklearn-pandas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - python
```
Comment (synapticarbors) [inline recipes/sklearn-pandas/meta.yaml:22]: Change `python` to `python >=3.6` here and under `run`.

**Comment by FernandezMathieu** [comment]
@synapticarbors Fixed, thanks for the review

**Comment by FernandezMathieu** [comment]
I think we can merge here

**Pull request merged**